Tilers/CityTiler/kd_tree.py

```diff
@@ -5,7 +5,7 @@ def kd_tree(cityobjects, maxNumCityMCityObjects, depth=0):
     # The module argument of 2 (in the next line) hard-wires the fact that
     # this kd_tree is in fact a 2D_tree.
     axis = depth % 2
-
+    
```

Comment:
Unnecessary spaces introduced.
Run some pep8 linter e.g. pylint on the files you modified/improved.

Reply:
Fixed by CCO